README.md

```diff
@@ -1,33 +1,72 @@
-# atlas
+# dolphin
 
 Algorithms and Tools for LAnd Surface Deformation using InSAR
 
 🚨 This toolbox is still in **pre-alpha** stage and undergoing **rapid development**. 🚨
 
-## Install
 
-The following will install `atlas` into a conda environment.
 
-1. Download source code:
+## Usage
+
+<!-- This is for snippets to copy sections into other documentation pages: -->
+<!-- https://facelessuser.github.io/pymdown-extensions/extensions/snippets/#snippet-sections -->
+# --8<-- [start:usage]
+Dolphin has a main command line entry point to run the algorithms and tools in workflows.
+The main entry point is named `dolphin`, which has two subcommands:
+
+1. `dolphin config`: create a workflow configuration file.
+2. `dolphin run` : run the workflow using this file.
+
+Example usage:
 
 ```bash
-git clone https://github.com/opera-adt/atlas.git && cd atlas
+$ dolphin config --slc-directory /path/to/slc --ext ".tif"
+```
+This will create a YAML file (by default `dolphin_config.yaml` in the current directory).
+You can also directly use a list of SLC files as input, e.g.:
+```bash
+$ dolphin config --slc-files /path/to/slc1.tif /path/to/slc2.tif
 ```
 
-2. Install dependencies:
+# --8<-- [end:usage]
+
+
+
 
+# --8<-- [start:install]
+## Install
+
+The following will install `dolphin` into a conda environment.
+
+1. Download source code:
+```bash
+git clone https://github.com/opera-adt/dolphin.git && cd dolphin
+```
+2. Install dependencies:
+```bash
+# assuming that mamba is installed: https://mamba.readthedocs.io/en/latest/
+# if not, start with:
+# conda install mamba -n base -c conda-forge
+mamba install -c conda-forge --file requirements.txt
+```
+3. Install `dolphin` via pip:
 ```bash
-conda install -c conda-forge --file requirements.txt
+# -e installs in development mode
+python -m pip install -e .
 ```
 
-3. Install `atlas` via pip:
+For development:
 
 ```bash
-# run "pip install -e" to install in development mode
-python -m pip install .
-```
+# run "pip install -e" to install with extra development requirements
+python -m pip install -e .[docs]
```

Comment:
(noting here in case others come by wondering the same): it's for optional dependencies https://setuptools.pypa.io/en/latest/userguide/dependency_management.html#optional-dependencies

I've changed this to have both `[docs]` and `[test]` based on our conversation